Command and Control/Network communication disovery

```diff
@@ -0,0 +1,18 @@
+// Query 1 shows you any network communication happened from endpoints to an specific Remote IP or Remote URL
+// Ensure to update RemoteIP and RemoteURL variable.
+// For questions @MiladMSFT on Twitter or milad.aslaner@microsoft.com by email
+NetworkCommunicationEvents
+| where EventTime > ago(30d)
+| where RemoteIP == "IP ADDRESS GOES HERE"
+or RemoteUrl contains "URL GOES HERE"
+| project EventTime, ComputerName, ActionType, RemoteIP, RemoteUrl, InitiatingProcessFileName, InitiatingProcessCommandLine
+
+// Query 2 shows you any network communication that happened from endpoints through a specific file to an Remote IP or Remote URL
+//Ensure to update RemoteIP, RemoteURL and InitatingProcessFileName
+// For questions @MiladMSFT on Twitter or milad.aslaner@microsoft.com by email
+NetworkCommunicationEvents
+| where EventTime > ago(30d)
+| where RemoteIP == " IP ADDRESS GOES HERE"
+or RemoteUrl contains " URL GOES HERE"
+and InitiatingProcessFileName == “FILENAME.EXE”
+| project EventTime, ComputerName, ActionType, RemoteIP, RemoteUrl, InitiatingProcessFileName, InitiatingProcessCommandLine
```

Comment:
consider adding more queries to extend footprint on top of downloads as well (FileOriginUrl and FileOriginReferrer in FileCreateEvents for downloads and MiscEvents for blocks) and/or on top of network addresses seen in block events - e.g. by smartscreen (query from MiscEvents table).
Can have a single query to rule them all using find, or add a few more queries, or don't add anything at this stage - also fine.

Reply:
I like the idea. I can add a 3rd query option that would essentially search for specific connection across multiple tables. Good?

Reply:
great. :)